src/nora/components/Header/Header.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './Header.module.scss'
+
+/**
+ * @private
+ *
+ * @param {JSX.Element} props.children - left children
+ */
+const Left = ({ children }) => {
+  return <div className={styles.Left}>{children}</div>
+}
+
+/**
+ * @private
+ *
+ * @param {JSX.Element} props.children - left children
+ */
+const Right = ({ children }) => {
+  return <div className={styles.Right}>{children}</div>
+}
+
+const Navbar = ({ children }) => {
+  return <nav className={styles.Navbar}>{children}</nav>
```

Comment:
After doing a bit of research, doing `<header><nav>...</nav></header>` feels most semantic to me.